src/components/AsyncCopyIconWithAddress/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Flex from 'antd/es/flex';
+import Avatar from 'antd/es/avatar';
+import { useDispatch, useSelector } from 'react-redux';
+import CopyIconWithAddress from '../CopyIconWithAddress';
+import { identityActions } from '../../redux/actions';
+import { identitySelectors } from '../../redux/selectors';
+import { decodeAndFilter } from '../../utils/identityParser';
+import { getAvatarParameters } from '../../utils/avatar';
+
+function AsyncCopyIconWithAddress({
+  address,
+  isTruncate,
+}) {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(identityActions.getIdentityOf.call(address));
+  }, [dispatch, address]);
```

Comment:
No. Purge this from orbit

Use getIdentitiesNames from noderpc call or something...

Reply:
Purged